recipes/nsimd/2.x/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class NsimdConan(ConanFile):
+    name = "nsimd"
+    homepage = "https://github.com/agenium-scale/nsimd"
+    description = "Agenium Scale vectorization library for CPUs and GPUs"
+    topics = ("hpc", "neon", "cuda", "avx", "simd", "avx2", "sse2", "aarch64", "avx512", "sse42", "rocm", "sve", "neon128")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
```

Comment:
Actually, fPIC=False won't be honored due to https://github.com/agenium-scale/nsimd/blob/v2.2/CMakeLists.txt#L154

Reply:
Now removing that line in CMakeLists.txt which I believe is sufficient to pick up Conan's setting.